data/json/mapgen/nested/house_nested.json

```
@@ -1546,5 +1546,633 @@
+      "place_items": [ { "chance": 80, "item": "creepy", "x": [ 1, 3 ], "y": [ 1, 3 ] } ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "compass direction indicates location of door",
+    "nested_mapgen_id": "bathroom_3x2_NW",
+    "object": {
+      "mapgensize": [ 3, 3 ],
+      "rotation": [ 0, 3 ],
+      "rows": [
+        ",,t",
+        "j~{",
+        "   "
+      ],
+      "palettes": [ "house_w_nest_palette" ],
+      "items": {
+        "t": { "item": "SUS_toilet", "chance": 80 },
+        "J": { "item": "SUS_hair_drawer", "chance": 20, "repeat": [ 1, 2 ] },
+        "j": [ { "item": "SUS_bathroom_sink", "chance": 50, "repeat": [ 1, 2 ] }, { "item": "SUS_bathroom_medicine", "chance": 50, "repeat": [ 1, 2 ] }, ],
+        "~": { "item": "SUS_bathroom_cabinet", "chance": 40, "repeat": [ 1, 2 ] }
+      }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "compass direction indicates location of door",
+    "nested_mapgen_id": "bathroom_3x2_NW2",
+    "object": {
+      "mapgensize": [ 3, 3 ],
+      "rotation": [ 0, 3 ],
+      "rows": [
+        ",,{",
+        "tj~",
+        "   "
+      ],
+      "palettes": [ "house_w_nest_palette" ],
+      "items": {
+        "t": { "item": "SUS_toilet", "chance": 80 },
+        "J": { "item": "SUS_hair_drawer", "chance": 20, "repeat": [ 1, 2 ] },
+        "j": [ { "item": "SUS_bathroom_sink", "chance": 50, "repeat": [ 1, 2 ] }, { "item": "SUS_bathroom_medicine", "chance": 50, "repeat": [ 1, 2 ] }, ],
+        "~": { "item": "SUS_bathroom_cabinet", "chance": 40, "repeat": [ 1, 2 ] }
+      }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "compass direction indicates location of door",
+    "nested_mapgen_id": "bathroom_3x2_NW3",
+    "object": {
+      "mapgensize": [ 3, 3 ],
+      "rotation": [ 0, 3 ],
+      "rows": [
+        ",,t",
+        "j~{",
+        "   "
+      ],
+      "palettes": [ "house_w_nest_palette" ],
+      "items": {
+        "t": { "item": "SUS_toilet", "chance": 80 },
+        "J": { "item": "SUS_hair_drawer", "chance": 40, "repeat": [ 1, 2 ] },
+        "j": [ { "item": "SUS_bathroom_sink", "chance": 50, "repeat": [ 1, 2 ] }, { "item": "SUS_bathroom_medicine", "chance": 50, "repeat": [ 1, 2 ] }, ],
+        "~": { "item": "SUS_bathroom_cabinet", "chance": 60, "repeat": [ 1, 2 ] }
+      },
+      "place_monsters": [ { "monster": "GROUP_PEST", "chance": 50, "x": 1, "y": 1 } ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "compass direction indicates location of door",
+    "nested_mapgen_id": "bathroom_3x2_NW4",
+    "object": {
+      "mapgensize": [ 3, 3 ],
+      "rotation": [ 0, 3 ],
+      "rows": [
+        ",,{",
+        "tj~",
+        "   "
+      ],
+      "palettes": [ "house_w_nest_palette" ],
+      "items": {
+        "t": { "item": "SUS_toilet", "chance": 80 },
+        "J": { "item": "SUS_hair_drawer", "chance": 40, "repeat": [ 1, 2 ] },
```

```suggestion
        "J": { "item": "SUS_hair_drawer", "chance": 40 },
```
This is one that goes through this whole nsted map section: SUS groups should probably not repeat unless you're totally sure. This suggests there is a significant chance of finding a bathroom cabinet with two hair drawers. If you want a chance of duplicating items, write that into the item group, not the mapgen spawn.